source/Outfit.cpp

```diff
@@ -261,6 +261,20 @@ void Outfit::Load(const DataNode &node)
 			cost = child.Value(1);
 		else if(child.Token(0) == "mass" && child.Size() >= 2)
 			mass = child.Value(1);
+		else if (child.Token(0) == "bunks" && child.Size() >= 2)
+		{
+			bunks = child.Value(1);
+			for(const DataNode &grand : child)
+			{
+				if(grand.Size() >= 2)
+				{
+					string loadBunkType = grand.Token(0);
+					pair<bool, pair<bool, bool>> bunkTypeData = GameData::BunkType(loadBunkType);
+					if(bunkTypeData.first)
+						bunkTypes[loadBunkType] = grand.Value(1);
```

Comment:
See this is what I mean with the comment above. This is not easy to understand for anyone reading the code; which is why its best not to use unnamed data.

Reply:
I will do this